packages/desktop-activity/src/lib/kb-mouse-timer.ts

```
@@ -0,0 +1,60 @@
+export class KbMouseTimer {
+	private static instance: KbMouseTimer;
+
+	private flushIntervalSeconds: number = 60;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)